res/values-de/strings.xml

```diff
@@ -0,0 +1,386 @@
+    <string name="issue_milestone_hint">Meilenstein</string>
+    <string name="issue_assignee_hint">Beauftragter</string>
+    <string name="link_not_openable">Der Link konnte nicht geöffnet werden</string>
+    <string name="download_created">%1$s wurde erstellt</string>
+    <string name="download_mobile_warning_title">Mobilfunkverbindung</string>
+    <string name="download_mobile_warning_message">Möchten Sie die Datei über ihre Mobilfunkverbindung herunterladen?\nSie können den Download auch ausführen, sobald eine WLAN-Verbindung besteht.</string>
+    <string name="download_now_button">Jetzt herunterladen</string>
+    <string name="download_wifi_button">Über WLAN herunterladen</string>
+    <string name="download_fail_no_storage_toast">Die Datei konnte nicht heruntergeladen werden - der interne Speicher oder die SD-Karte ist nicht verfügbar.</string>
+	<string name="auth_code">Bestätigungscode</string>
+	<string name="two_factor_auth">2-Faktor-Authentifizierung</string>
+	<string name="verify">Überprüfen</string>
+    <string name="confirmation">Sind Sie sicher?</string>
+    <string name="unset">Aufheben</string>
+    <string name="logout_pref_summary_logged_in">Zur Zeit als %1$s angemeldet</string>
+    <string name="logout_pref_summary_logged_out">Zur Zeit nicht angemeldet</string>
+    <string name="copyright_notice">\u00a9 2014 Azwan Adli, Danny Baumann</string>
+    <string name="actions">Aktivitäten</string>
+    <string name="created">Erstellt</string>
+    <string name="assigned">Zugewiesen</string>
+    <string name="mentioned">Erwähnt</string>
+    <string name="my_issues">Eigene Probleme</string>
+    <string name="my_pull_requests">Eigene Pull-Requests</string>
+
+    <plurals name="contributor_extra_data">
+        <item quantity="one">%1$d Beitrag</item>
+        <item quantity="other">%1$d Beiträge</item>
+    </plurals>
+
+    <string name="repo_type_all">Alle</string>
+    <string name="repo_type_owner">Eigentümer</string>
+    <string name="repo_type_member">Mitglied</string>
+    <string name="repo_type_public">Öffentlich</string>
+    <string name="repo_type_private">Privat</string>
+    <string name="repo_type_sources">Quellen</string>
+    <string name="repo_type_forks">Forks</string>
+    <string name="repo_type_watched">Verfolgte</string>
+    <string name="repo_type_starred">Favorisierte</string>
+
+    <string name="no_blogs_found">Es wurden keine Blogs gefunden.</string>
+    <string name="no_comments_found">Bis jetzt hat niemand kommentiert.</string>
+    <string name="no_files_found">Es wurden keine Dateien gefunden.</string>
+    <string name="no_trends_found">Zur Zeit sind keine Repositories angesagt.</string>
+    <string name="no_downloads_found">Diese Repository enthält keine Downloads.</string>
+    <string name="no_milestones_found">Diese Repository hat keine Meilensteine konfiguriert.</string>
+    <string name="no_commits_found">Es wurden keine Beiträge gefunden.</string>
```

Comment:
`Commit` dürfte tatsächlich besser sein, da es in dem Zusammenhang ein Fachbegriff ist...